<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjustesSensores</class>
 <widget class="QDialog" name="AjustesSensores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajustes de Sensores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/IMU.png</normaloff>:/images/IMU.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:rgb(221, 221, 221)
}

QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Ajustes de captura sensores</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelDivisorFrecuenciaMuestreo">
        <property name="text">
         <string>Frecuencia de Muestreo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelRangoAcelerometro">
        <property name="text">
         <string>Rango Acelerometro</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxAscale"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelRangoGiroscopio">
        <property name="text">
         <string>Rango Giroscopio</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxGscale"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelFiltroPasaBajo">
        <property name="text">
         <string>Filtro Pasa-Bajo Digital</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxFrecuenciaMuestreo"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxDLPF"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelSampleRate">
       <property name="text">
        <string>Frecuencia de Muestreo: ?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDLPF">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Filtro Pasa-Bajo= ?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelAdvertencia">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Advertencia: No se asegura un correcto muestreo sobre 275 Hz.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: rgb(255, 0, 0);
border: 2px solid;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: rgb(170, 0, 0);
border: 2px solid ;
}

QPushButton:pressed {
border: 3px solid rgb(102, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
